packages/react/select/src/select.tsx

```diff
@@ -1659,26 +1663,35 @@ const BubbleSelect = React.forwardRef<HTMLSelectElement, React.ComponentPropsWit
     }, [prevValue, value]);
 
     /**
-     * We purposefully use a `select` here to support form autofill as much
-     * as possible.
+     * We purposefully use a `select` here to support form autofill as much as
+     * possible.
      *
      * We purposefully do not add the `value` attribute here to allow the value
-     * to be set programmatically and bubble to any parent form `onChange` event.
-     * Adding the `value` will cause React to consider the programmatic
+     * to be set programmatically and bubble to any parent form `onChange`
+     * event. Adding the `value` will cause React to consider the programmatic
      * dispatch a duplicate and it will get swallowed.
      *
-     * We use `VisuallyHidden` rather than `display: "none"` because Safari autofill
-     * won't work otherwise.
+     * We use visually hidden styles rather than `display: "none"` because
+     * Safari autofill won't work otherwise.
      */
     return (
-      <VisuallyHidden asChild>
-        <select {...selectProps} ref={composedRefs} defaultValue={value} />
-      </VisuallyHidden>
+      <Primitive.select
+        {...props}
+        style={{ ...VISUALLY_HIDDEN_STYLES, ...props.style }}
```

Comment:
Very minor change, but if we can just expose the style object it removes a tiny bit of overhead and excess noise in the component tree (which is nicer for dev tools IMO)